tickets/k8s/proxyNode_v1.org

```diff
@@ -0,0 +1,338 @@
+   audit_id                   | text    |           | not null | 
+   stage                      | text    |           | not null | 
+   event_verb                 | text    |           | not null | 
+   request_uri                | text    |           | not null | 
+   operation_id               | text    |           |          | 
+   event_level                | text    |           |          | 
+   event_stage                | text    |           |          | 
+   api_version                | text    |           |          | 
+   useragent                  | text    |           |          | 
+   test_hit                   | boolean |           |          | 
+   conf_test_hit              | boolean |           |          | 
+   event_user                 | jsonb   |           |          | 
+   object_namespace           | text    |           |          | 
+   object_type                | text    |           |          | 
+   object_group               | text    |           |          | 
+   object_ver                 | text    |           |          | 
+   source_ips                 | jsonb   |           |          | 
+   annotations                | jsonb   |           |          | 
+   request_object             | jsonb   |           |          | 
+   response_object            | jsonb   |           |          | 
+   response_status            | jsonb   |           |          | 
+   stage_timestamp            | text    |           |          | 
+   request_received_timestamp | text    |           |          | 
+   data                       | jsonb   |           | not null | 
+  Indexes:
+      "idx_audit_event_bucket" btree (bucket)
+      "idx_audit_event_conf_test_hit" btree (conf_test_hit)
+      "idx_audit_event_job" btree (job)
+      "idx_audit_event_operation_id" btree (operation_id)
+      "idx_audit_event_test_hit" btree (test_hit)
+  Triggers:
+      add_op_id BEFORE INSERT ON audit_event FOR EACH ROW WHEN (new.job = 'live'::text) EXECUTE FUNCTION add_op_id()
+
+  #+end_SRC
+
+** look at what tilt does
+
+  #+begin_src sql-mode :eval never-export :exports both :session none
+    select distinct useragent
+      from audit_event
+     where bucket='apisnoop'
+       and useragent not like 'kube%'
+       and useragent not like 'coredns%'
+       and useragent not like 'kindnetd%'
+      -- and useragent like 'live%'
+       and operation_id in (
+         connectCoreV1PutNodeProxyWithPath,
+         connectCoreV1PutNodeProxy,
+         connectCoreV1PostNodeProxyWithPath,
+         connectCoreV1PostNodeProxy,
+         connectCoreV1PatchNodeProxyWithPath,
+         connectCoreV1PatchNodeProxy,
+         connectCoreV1OptionsNodeProxyWithPath,
+         connectCoreV1OptionsNodeProxy,
+         connectCoreV1HeadNodeProxyWithPath,
+         connectCoreV1HeadNodeProxy,
+         connectCoreV1GetNodeProxy,
+         connectCoreV1DeleteNodeProxyWithPath,
+         connectCoreV1DeleteNodeProxy
+         )
+        ;
```

Comment:
@zachmandeville Can you help me find the correct for this query

Reply:
The list of values in your where claus `operation id in ()` are being read as expected columns in the audit_event table.  If you surround each one with `'` they'll be read as strings, as you intended.

I would recommend running the query again without this clause, to verify that you are getting useragents returned.